kubernetes/linera-validator/build_and_redeploy.sh

```diff
@@ -28,12 +34,15 @@ usage() {
 handle_options() {
     while [ $# -gt 0 ]; do
         case $1 in
-        -h | --help) usage; exit 0;;
+        -h | --help)
+            usage
+            exit 0
+            ;;
         --cloud) cloud_mode=1;;
-        --port-forward) port_forward=1;;
-        --no-build) do_build=;;
-        --clean) clean=1;;
-        --copy) copy=1;;
+        --port-forward) port_forward=1 ;;
```

Comment:
I'm confused what happened here — it wants a space for line 42 but not line 41?  Why?

Reply:
It probably only keeps it in one line if there's a single command in it

Reply:
Both are still one line, but on line 42 it demands that we add a space before the `;;`, while on line 41 it seems completely happy to leave the `;;` without a space.  The structure of the lines is more or less identical, so I'm confused as to what rule it used to decide :thinking:

Reply:
Ah, I misunderstood what you said. Yeah, the space there but not on line 41 seems a bit arbitrary 🤔